src/Twig/StimulusTwigExtension.php

```diff
@@ -73,6 +73,79 @@ public function renderStimulusController(Environment $env, $dataOrControllerName
 
+    /**
+     * @param string|array $dataOrControllerName This can either be a map of controller names
+     *                                           as keys set to their "actions" and "events".
+     *                                           Or this can be a string controller name and
+     *                                           action and event are passed as the 2nd and 3rd arguments.
+     * @param string|null  $actionName The action to trigger if a string is passed to the 1st argument. Optional.
+     * @param string|null  $eventName  The event to listen to trigger if a string is passed to the 1st argument. Optional.
+     *
+     * @return string
+     * @throws \Twig\Error\RuntimeError
+     */
+    public function renderStimulusAction(Environment $env, $dataOrControllerName, string $actionName = null, string $eventName = null): string
+    {
+        if (is_string($dataOrControllerName)) {
+            $data = [$dataOrControllerName => $eventName === null ? [[$actionName]] : [[$eventName => $actionName]]];
+        } else {
+            if ($actionName || $eventName) {
+                throw new \InvalidArgumentException('You cannot pass a string to the second or third parameter while passing an array to the first argument of stimulus_action(): check the documentation.');
+            }
+
+            $data = $dataOrControllerName;
+
+            if (!$data) {
+                return '';
+            }
+        }
+
+        $actions = [];
+
+        foreach ($data as $controllerName => $controllerActions) {
+            $controllerName = twig_escape_filter($env, $this->normalizeControllerName($controllerName), 'html_attr');
+
+            if (is_string($controllerActions)) {
+                $controllerActions = [[$controllerActions]];
+            }
+
+            foreach ($controllerActions as $possibleEventName => $controllerAction) {
+                if (is_string($possibleEventName) && is_string($controllerAction)) {
+                    $controllerAction = [$possibleEventName => $controllerAction];
+                } else if (is_string($controllerAction)) {
+                    $controllerAction = [$controllerAction];
+                }
+
+                foreach ($controllerAction as $eventName => $actionName) {
+                    $action = $controllerName.'#'.twig_escape_filter($env, $actionName, 'html_attr');
+
+                    if (is_string($eventName)) {
+                        $action = $eventName.'->'.$action;
+                    }
+
+                    $actions[] = $action;
+                }
+            }
+        }
+
+        return rtrim('data-action="'.implode(' ', $actions).'"');
+    }
+
+    /**
+     * @param string $controllerName The controller name.
+     * @param string $targetName The target name.
+     *
+     * @return string
+     * @throws \Twig\Error\RuntimeError
+     */
+    public function renderStimulusTarget(Environment $env, string $controllerName, string $targetName): string
```

Comment:
Given that the other two helper functions allow us to pass in an array, would it be a more consistent user-experience if this functioned in the same way?

Reply:
Do you mean something like this?
```twig
{{ stimulus_target({
  'controller': 'target1 target2',
  'controller-2': 'target3 target4',
}) }}
```

Reply:
Yes, exactly, that would be consistent with the other helpers.

Reply:
Added in 446be773b5c1f0c903e9fc4c01d221dc99446d17